Cargo.toml

```diff
@@ -21,6 +21,7 @@ rangemap = "1.4.0"
 rustc-hash = { version = "1.1.0", default-features = false }
 rustybuzz = { version = "0.14", default-features = false, features = ["libm"] }
 self_cell = "1.0.1"
+smol_str = { version = "0.2.2", default-features = false }
```

Comment:
For what it's worth, `smol_str` doesn't seem to be recommended anymore (see the comment by the original author in https://www.reddit.com/r/rust/comments/117ksvr/comment/j9eh35d/)

There's also a comparison of small string crates at https://github.com/rosetta-rs/string-rosetta-rs?tab=readme-ov-file (`smol_str` was removed from the table in favor of ecow)